client/router_update.go

```diff
@@ -1000,6 +1000,14 @@ func (cli *VanClient) createClaimsServerSecret(ctx context.Context, namespace st
 }
 
 func (cli *VanClient) createClaimsRedemptionRoute(ctx context.Context, namespace string) error {
+	siteConfig, err := cli.SiteConfigInspect(ctx, nil)
+	if err != nil {
+		return err
+	}
+	host := siteConfig.Spec.GetRouterIngressHost()
+	if host != "" {
+		host = types.ClaimRedemptionRouteName + "-" + namespace + "." + host
+	}
```

Comment:
Just as a philosophical point, I don't think we needed to do anything here as this is only used when upgrading from an old version where there was no expectation that the host could be set on routes. That said I can't see that it will cause any harm so fine to leave it.

Reply:
Agree, will remove